test/conformance/kubetestgen/types.go

```diff
@@ -35,5 +35,7 @@ type Behavior struct {
 	APIObject   string `json:"apiObject,omitempty"`
 	APIField    string `json:"apiField,omitempty"`
 	APIType     string `json:"apiType,omitempty"`
+	Version     string `json:"version,omitempty"`
```

Comment:
Should we prepend v1.18 (or whatever the version should be) to all the behaviors we're adding?

Reply:
Hmm. Good question. Most of these behaviors have been around for a long, long time. But once we transition to the behavior model, then it will really only apply to those newer versions. I had put in v1.14 for the readiness gates; that's when it went GA. But it would be easier to just make them all v1.18, since no earlier cluster would be subject to these rules.

Reply:
Should I just remove API* fields? They were intended to provide context / help with regeneration when using `kubetestgen`. But at least for now we're not really re-generating things, but instead just using them for bootstrapping. Different PR though.

Reply:
I think it's helpful in certain situations when we're just enumerating over one field (eg service type)...although we should probably be more explicit in specifying which fields are mandatory vs optional.

Different PR, I agree.

Reply:
I ended up removing version and status for now, we can deal with that later. I also simplified the initial lists of behaviors.